.circleci/config.yml

```diff
@@ -1,23 +1,31 @@
-defaults: &defaults
+checkout_submodule: &checkout_submodule
+  run:
+    name: Checkout submodule
+    command: |
+        git submodule sync
+        git submodule update --init --recursive
+
+npm_cache_key: &npm_cache_key
+  npm-deps-v1-{{ checksum "package.json" }}
```

Comment:
Cool!  Why the `-v1-`?

Reply:
@kevinbarabash It's just for internal versioning of the cache, allowing to invalidate the cache.